validator/src/main/java/com/google/daq/mqtt/validator/ReportingDevice.java

```diff
@@ -201,12 +201,24 @@ public void validateMessageType(Object message, Date timestamp, Map<String, Stri
     } else if (!missingPoints.isEmpty()) {
       addError(pointValidationError("missing points", missingPoints), attributes,
           Category.VALIDATION_DEVICE_CONTENT);
+      System.err.println(
+          String.format(
+              "Device has missing points: %s",
+              Joiner.on(", ").join(missingPoints)
```

Comment:
nit: there's com.google.udmi.util.GeneralUtils.CSV_JOINER for this... very minor savings in characters :-)